src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -6249,4 +6249,38 @@ static void Main(string[] args)
             AssertFormat(code, code);
         }
     }
+
+    [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+    public void NewLinesForBracesInPropertiesTest()
+    {
+        var changingOptions = new Dictionary<OptionKey, object>();
+        changingOptions.Add(CSharpFormattingOptions.NewLinesForBracesInProperties, false);
+        AssertFormat(@"class Class2
+{
+    int Foo
+    {
+        get
+        {
+            return 1;
+        }
+    }
+
+    int MethodFoo()
+    {
+        return 42; 
+    }
+}", @"class Class2
+{
+    int Foo {
```

Comment:
In theory yes - but the roslyn formatter isn't very fine grained and has a very basic option model. 
For example it only supports 2 brace styles.
Therefore I only introduced one option for property and accessor - since that matches the mono and c# k&r coding style.